node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/coredump/CoredumpHandler.java

```diff
@@ -175,6 +184,7 @@ String getMetadata(NodeAgentContext context, ContainerPath coredumpDirectory, Su
                     .resolve(context.containerName().asString())
                     .resolve(coredumpDirectory.getFileName().toString())
                     .resolve(coredumpFile.getFileName().toString()).toString());
+            decryptionToken.ifPresent(token -> metadata.put("decryption_token", token));
```

Comment:
This JSON is sent to a Vespa application, so need to add this field there (Not an issue for this PR since this is not enabled yet, just reminder :smile: )

Reply:
Thanks, I'll make sure to update the schema prior to wiring this in anywhere.